### PR TITLE
Formatted Next 15 Files

### LINKED ISSUE #4109: Apply black formatter
xref https://github.com/pymc-devs/pymc3/pull/4108#issuecomment-694462527

Reviewing PRs which change code _and_ change formatting can be tricky, so my suggestion is to apply `black` to batches of 10-15 files at a time, merge those in, and then add `black` to `.pre-commit-config.yaml`.

This would make for a good beginner-friendly issue. Make sure you're using `black` version 20.8b1 for this.

See  #4110 for an example of how to open a PR for this

EDIT
----

have removed the list of files as it looks like existing PRs might be enough to close this off. Please see #4095 for a similar issue to tackle

EDIT2
----

As of 2020-10-08, existing PRs should be enough to address all files. To close this issue, a pre-commit hook should be added

### PR DESCRIPTION
**Pull request by Yash1256**
Issue #4109 Applying Formatting on Next 15 Files
1. pymc3/tests/test_math.py
2. pymc3/tests/test_model_func.py
3. pymc3/tests/test_model_graph.py
4. pymc3/tests/test_examples.py
5. pymc3/tests/test_model_helpers.py
6. pymc3/tests/test_modelcontext.py
7. pymc3/tests/test_pickling.py
8. pymc3/tests/test_models_utils.py
9. pymc3/sampling.py
10. pymc3/model.py
11. pymc3/tests/test_posdef_sym.py
12. pymc3/tests/test_posterior_predictive.py
13. pymc3/tests/test_models_linear.py
14. pymc3/tests/test_parallel_sampling.py
15. pymc3/tests/test_minibatches.py

### CODE REVIEW

File: pymc3/model.py [hunk 1]
```diff
@@ -529,9 +524,7 @@ def tree_contains(self, item):
 
     def __setitem__(self, key, value):
         raise NotImplementedError(
-            "Method is removed as we are not"
-            " able to determine "
-            "appropriate logic for it"
+            "Method is removed as we are not" " able to determine " "appropriate logic for it"
```
Comment (MarcoGorelli) [inline pymc3/model.py]:
```suggestion
            "Method is removed as we are not able to determine appropriate logic for it"
```

File: pymc3/model.py [hunk 2]
```diff
@@ -1625,16 +1603,13 @@ def _get_scaling(total_size, shape, ndim):
...
-        ]
+        begin_coef = [floatX(t) / shp_begin[i] for i, t in enumerate(begin) if t is not None]
         end_coef = [floatX(t) / shp_end[i] for i, t in enumerate(end) if t is not None]
         coefs = begin_coef + end_coef
         coef = tt.prod(coefs)
     else:
         raise TypeError(
-            "Unrecognized `total_size` type, expected "
-            "int or list of ints, got %r" % total_size
+            "Unrecognized `total_size` type, expected " "int or list of ints, got %r" % total_size
```
Comment (MarcoGorelli) [inline pymc3/model.py]:
```suggestion
            "Unrecognized `total_size` type, expected int or list of ints, got %r" % total_size
```

**Comment by MarcoGorelli** [comment]
Nice, thanks, there's just a couple of places where `black` joins concatenates strings implicitly on the same line so I've suggested changes there

**Comment by Yash1256** [comment]
@MarcoGorelli Sir in the file model.py total 4 occurrence's of `" "` were there so I replaced all the 4 with space.

File: pymc3/tests/test_examples.py [hunk 3]
```diff
@@ -148,30 +153,138 @@ def too_slow(self):
...
 
 def build_disaster_model(masked=False):
-    disasters_data = np.array([4, 5, 4, 0, 1, 4, 3, 4, 0, 6, 3, 3, 4, 0, 2, 6,
-                               3, 3, 5, 4, 5, 3, 1, 4, 4, 1, 5, 5, 3, 4, 2, 5,
-                               2, 2, 3, 4, 2, 1, 3, 2, 2, 1, 1, 1, 1, 3, 0, 0,
-                               1, 0, 1, 1, 0, 0, 3, 1, 0, 3, 2, 2, 0, 1, 1, 1,
-                               0, 1, 0, 1, 0, 0, 0, 2, 1, 0, 0, 0, 1, 1, 0, 2,
-                               3, 3, 1, 1, 2, 1, 1, 1, 1, 2, 4, 2, 0, 0, 1, 4,
-                               0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0, 1])
+    disasters_data = np.array(
```
Comment (AlexAndorra) [inline pymc3/tests/test_examples.py]: Please use `# fmt: off` here, to disable (and then re-enable) Black formatting -- see step 3 of the [style guide](https://github.com/pymc-devs/pymc3/wiki/PyMC's-Jupyter-Notebook-Style)

**Review by AlexAndorra — CHANGES_REQUESTED** [review]
Thanks @Yash1256 ! Almost all good: just one place where Black formatting should be disabled to preserve readability (see my comment below)

File: pymc3/tests/test_examples.py (hunk 3 above)
Comment (MarcoGorelli) [reply pymc3/tests/test_examples.py]: @Yash1256 thanks for updating - I think you'll want to checkout this file to how it is on upstream/master before putting `# fmt: off` and running `black` again.

Workflow would be something like
```
git fetch upstream --prune
git checkout upstream/master -- pymc3/tests/test_examples.py
<add in #fmt: off and #fmt: on as suggested by Alex>
black pymc3/tests/test_examples.py
git add -u
git commit -m <some descriptive message lol>
git push -u origin HEAD
```

Comment (Yash1256) [reply pymc3/tests/test_examples.py]: Please review @AlexAndorra @MarcoGorelli

**Review by MarcoGorelli — APPROVED** [review]
Nice!

**Review by AlexAndorra — APPROVED** [review]
All good now, thanks @Yash1256 !

**Pull request merged**